libnavigation-ui/src/main/java/com/mapbox/navigation/ui/legacy/RouteUtils.kt

```diff
@@ -4,7 +4,6 @@ import com.mapbox.api.directions.v5.models.BannerInstructions
 import com.mapbox.api.directions.v5.models.LegStep
 import com.mapbox.geojson.Point
 import com.mapbox.navigation.base.trip.model.RouteProgress
-import com.mapbox.navigation.base.trip.model.RouteProgressState
 import timber.log.Timber
 
 /**
```

Comment:
Let's remove the `RouteUtils` altogether for now. What do you think? Currently is never used.

Reply:
sounds good to me. I am kind of hesitate to so because we might are lacking some features somewhere which needs the rest of the methods.

but we could remove it now and bring it back when necessary.

Reply:
![yagni](https://user-images.githubusercontent.com/1668582/81344496-d90fde00-9084-11ea-887b-6bf2fa20b1aa.jpeg)